skiko/src/iosMain/kotlin/org/jetbrains/skiko/redrawer/MetalRedrawer.ios.kt

```diff
@@ -51,6 +52,7 @@ internal class MetalRedrawer(
     override fun dispose() {
         if (!isDisposed) {
             frameDispatcher.cancel()
+            super.dispose()
```

Comment:
I got rid of abstract class, and made Redrawer interface again.

Now we call `contextHandler.dispose` here instead of `super.dispose`.

Where to cancel coroutine probably doesn't matter, as we either have a lock, or just call everything in Swing thread.